game/plugin/skills/prayer/src/data.kt

```diff
@@ -0,0 +1,75 @@
+import org.apollo.game.message.impl.ConfigMessage
+import org.apollo.game.model.Animation
+import org.apollo.game.model.entity.Mob
+import org.apollo.game.model.entity.Player
+
+val BURY_BONE_ANIMATION = Animation(827)
+
+val PLAYER_PRAYERS = hashMapOf<Player, ArrayList<Prayer>?>()
+
+fun Player.getCurrentPrayers(): ArrayList<Prayer> {
+    PLAYER_PRAYERS[this] = PLAYER_PRAYERS[this] ?: ArrayList<Prayer>(); //Make sure that the array list exists
```

Comment:
Any thoughts on a more generic way of handling these scenarios @Major- ?

Reply:
Ye we need some sort of replacement for the attributes system, this is fine for now.

Although I think it should be a `Set` not `ArrayList` - we don't need ordering, right?

Reply:
Good catch, yes it does not need ordering.